easybuild/easyconfigs/n/novaSTA/novaSTA-1.1-foss-2022a-CUDA-11.7.0.eb

```diff
@@ -0,0 +1,48 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2022/01
+easyblock = 'MakeCp'
+name = 'novaSTA'
+version = '1.1'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/turonova/novaSTA'
+description = """ C++ version of subtomogram averaging (SA) scripts from TOM/AV3 package          
+https://doi.org/10.1073/pnas.0409178102. Both CPU and GPU parallelization is    
+supported although the latter performs significantly worse in terms of          
+processing time (the code is not well optimized) and is thus not recommended for
+larger datasets.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {
+    'cstd': 'c++14',
+    'opt': 'True',
+    'debug': 'False',
+    'usempi': 'True'
+}
+
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['db0765eade1b0399929dcd61f2dbeeffd762885902f6ef2811d1fb4f9634a2ac']
+
+github_account = 'turonova'
+dependencies = [('CUDA', '11.7.0', '', SYSTEM)]
```

Comment:
Needs UCX-CUDA as dependency since this is MPI code with CUDA under a foss toolchain